### DISCUSSION TITLE
Datetime and indexes

### DISCUSSION THREAD

**Discussion by paulbrickellmaven** [prisma/prisma#13050] [Q&A] [answered]
Hi all I have a question about datetimes hopefully someone can give me some guidance.

I have a query that is selecting data based on a datetime column. This column is indexed but I am seeing poor performance when I query for data. In the explain plan for the query I see the generated query results in a filesort on the datetime column. The clause that gives me a problem looks like this (table and column names redacted)...

AND `table`.`column` < '2022-04-26T23:00:00.000Z'

If I modify the query to this...

AND `table`.`column` < cast('2022-04-26T23:00:00.000Z' as datetime)

In mysql workbench the explain plan now uses the index and I see an order of magnitude improvement in performance.

My question is, is there a way to stop Prisma from converting to ISO strings?

**Comment by pantharshit00** [comment, marked as answer]
Hello @paulbrickellmaven , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I don't think this is possible currently in Prisma. I will suggest you to open a feature request here: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
